agent/consul/state/state_store_test.go

```diff
@@ -57,17 +59,37 @@ func testStateStore(t *testing.T) *Store {
 	return s
 }
 
+func testRegisterPeerings(t *testing.T, s *Store, idx uint64, name string) *pbpeering.Peering {
+	uuid, err := uuid.GenerateUUID()
+	require.NoError(t, err)
+	peering := &pbpeering.Peering{Name: name, ID: uuid}
+	err = s.PeeringWrite(idx, peering)
+	require.NoError(t, err)
+	return peering
+}
+
+func testRegisterPeeringsWithPartition(t *testing.T, s *Store, idx uint64, name, partition string) *pbpeering.Peering {
```

Comment:
Ashwin moved this to `state_store_ent_test.go` in his last commit (which he force pushed). Maybe you may have cherrypicked an older commit?